setup.py

```diff
@@ -9,7 +9,7 @@
 description = "A collection of first party Google Cloud Platform event objects."
 version = "0.1.1"
 release_status = "Development Status :: 3 - Alpha"
-dependencies = []
+dependencies = ["python-dateutil==2.8.1"]
 extras = {
```

Comment:
I do believe the approach @eclipselu takes for this file to be more readable here:

https://github.com/googleapis/google-cloudevents-python/pull/29/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7

Could we merge the PRs and use that style?

---

If there is a reason for using this style, please state the reason in a comment and possibly link the requirement from a publishing script for example.

Reply:
I believe the format of the current `setup.py` comes from Bu Sun's PR (#15) when she set up the auto-release tool.

Reply:
If there's no mandate style preference, i would prefer inline style as setup.py is more of a configuration here. Another option is to put these static configurations in setup.cfg, like [what Flask does](https://github.com/pallets/flask/blob/master/setup.cfg)